src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Adapter/ReferenceEntity/ReferenceEntityAttributeOptionApiAdapterInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Adapter\ReferenceEntity;
+
+interface ReferenceEntityAttributeOptionApiAdapterInterface
+{
+    /**
+     * @param string $referenceEntityCode Code of the reference entity
```

Comment:
Params description is quite redundant as long as variables names quite descriptive